components/interactive_dialog/index.js

```diff
@@ -24,14 +26,18 @@ function mapStateToProps(state) {
         submitLabel: data.dialog.submit_label,
         notifyOnCancel: data.dialog.notify_on_cancel,
         state: data.dialog.state,
+        emojiMap: getEmojiMap(state)
     };
 }
 
 function mapDispatchToProps(dispatch) {
     return {
-        actions: bindActionCreators({
-            submitInteractiveDialog,
-        }, dispatch),
+        actions: bindActionCreators(
```

Comment:
Why change the formatting here? Was this intentional?

Reply:
no xD probably had a weird autoformatter